### ISSUE TITLE
Saved Image not Displayed.

### ISSUE THREAD

**Issue by ghost** [pyscript/pyscript#1970]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

I am trying to save an image.I think the image is saved and not showing up in my py-script project.
**display(os.listdir(s))**  shows that the image is present in the directory. **heading.jpg** is the image.
https://_12.pyscriptapps.com/empty-pine/latest/

### What browsers are you seeing the problem on? (if applicable)

Microsoft Edge

**Comment by WebReflection** [comment]
Please address @alex-martinez or @FabioRosado when you open website related issue as this repository is only for the PyScript runtime. I still think we should have a dedicated space for these issue but this time I'll ping them myself.

**Comment by nasrin1748** [comment]
Thanks for letting us know.

**Comment by nasrin1748** [comment]
I am able to save file using the following code in PyCharm but not in Py-script.                 
                        https://_12.pyscriptapps.com/delicate-pond/latest/

**Comment by WebReflection** [comment]
> I am able to save file using the following code in PyCharm

I am not sure how that works but I might guess their online IDE offers some way to persist files on their host ... which I still think we should be able to do at some point as this is the most requested feature these days ( /cc @mchilvers @FabioRosado )

For the time being though, we recently introduced the [pyscript.storage](https://docs.pyscript.net/2024.8.2/api/#pyscriptstorage) which will persists data on your browser (still not on your disk, it's still persistent across sessions and reboots).

If you convert an image into a buffer and store it, you can then retrieve it back any time you like.

I hope this works as workaround/solution for your needs, still I'd like to understand we're all on-board with the fact users want, and need, to be able to read and store files in PyScript .com.

/cc @ntoll

**Comment by FabioRosado** [comment]
> I am not sure how that works but I might guess their online IDE offers some way to persist files on their host ... which I still think we should be able to do at some point as this is the most requested feature these days ( /cc @mchilvers @FabioRosado )

100% we don't support saving files from the virtual filesystem yet, but we have talked in the past about the possibility to support this

**Comment by WebReflection** [comment]
@FabioRosado it doesn't have to be reflected from the vFS ... we could offer a different API to store live stuff which might be an extra module we provide on PyScript .com only that works with PyScript .com only too ... just brainstorming here, but I wouldn't mix vFS and PyScript .com persistent storage scope ... these are two very different use cases.

**Comment by FabioRosado** [comment]
> @FabioRosado it doesn't have to be reflected from the vFS ... we could offer a different API to store live stuff which might be an extra module we provide on PyScript .com only that works with PyScript .com only too ... just brainstorming here, but I wouldn't mix vFS and PyScript .com persistent storage scope ... these are two very different use cases.

Yeah I started working on such module, but I'm not sure the current status of this as I was moved to work on other stuff 🤔

**Comment by nasrinpathan** [comment]
Browser has security restrictions so it won't allow users the option to save. But if you are connected with desktop you have the option to select the folder where you want to save. Github is client-side so code doesn't work after I deploy. 
https://www.youtube.com/watch?v=UNq2vuvGVV0
